src/main/java/pl/pabilo8/immersiveintelligence/common/items/tools/ItemIILighter.java

```diff
@@ -47,6 +47,7 @@ public class ItemIILighter extends ItemIIBase implements ITool
 	public ItemIILighter()
 	{
 		super("lighter", 1);
+		LighterFuelHandler.addFuel(IEcontent.fluidPlantoil, 100);
```

Comment:
Please move it to the CommonProxy class, so it will be in the same place as other fuels. https://github.com/Pabilo8/ImmersiveIntelligence/blob/c0d05122f24689199f043db04f249b919f5fe11d/src/main/java/pl/pabilo8/immersiveintelligence/common/CommonProxy.java#L464

Reply:
aight